<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login_1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:1, y1:0.614, x2:1, y2:0, stop:0 rgba(85, 0, 127, 255), stop:1 rgba(255, 85, 255, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Roman 2&quot;;
color:white</string>
    </property>
    <property name="text">
     <string>Spin camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_User">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Roman 2&quot;;
color:white</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Roman 2&quot;;
color:white</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Contrase√±a:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_User">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgba(255, 85, 0, 50);
font: 13pt &quot;Roman 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ingresar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(0, 85, 255);
font: 13pt &quot;Roman 2&quot;;</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SALIR">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(0, 85, 255);
font: 13pt &quot;Roman 2&quot;;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_Password">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgba(255, 85, 0, 30);
font: 13pt &quot;Roman 2&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Roman 2&quot;;
color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_limpiar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(0, 85, 255);
font: 13pt &quot;Roman 2&quot;;</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
